modules/api/test/ApiTest.php

```diff
@@ -0,0 +1,388 @@
+<?php declare(strict_types=1);
+/**
+ * This contains tests relevant to the API
+ *
+ * PHP Version 7
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Login
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+
+namespace LORIS\api\Test;
+ini_set('memory_limit', '1024M');
+
+require './vendor/autoload.php';
+
+use PHPUnit\Framework\TestCase;
+use Laminas\Diactoros\ServerRequest;
```

Comment:
You should be using Laminas\Diactoros\Request because we are on the client side.